sql/core/src/main/scala/org/apache/spark/sql/util/QueryExecutionListener.scala

```diff
@@ -131,6 +127,14 @@ class ExecutionListenerManager private[sql](session: SparkSession, loadExtension
 private[sql] class ExecutionListenerBus(sessionUUID: String)
```

Comment:
I think you can make this constructor priviate.

Reply:
good idea!